docs/learn/learn-parachains.md

```diff
@@ -199,28 +174,72 @@ from the Relay Chain, allowing for more efficient parachain processing.
 teams can either bid with their own {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} tokens,
 or source them from the community using the [crowdloan functionality](learn-crowdloans.md).
 
-[Parathreads](learn-parathreads.md) have the same API as parachains, but are scheduled for execution
-on a pay-as-you-go basis with an auction for each block.
-
 ### Parachain Lease Expiration
 
 When a parachain wins an auction, the tokens it bids get reserved until the lease's end. Reserved
 balances are non-transferrable and cannot be used for staking. At the end of the lease, the tokens
-are unreserved. Parachains that have not secured a new lease to extend their slot will automatically
-become [parathreads](./learn-parathreads.md).
+are unreserved. Parachains that have not secured a new lease to extend their slot can acquire one
+[on-demand](#on-demand-parachains).
 
 ## System Parachains
 
-System parachains are parachains that use execution cores allocated by the network's governance.
-These chains remove transactions from the Relay Chain, allowing network validators to allocate
-resources to validating parachains. System chains are Polkadot using its scaling technology to host
-itself.
+[System parachains](./learn-system-chains.md) are parachains that use execution cores allocated by
+the network's governance. These chains remove transactions from the Relay Chain, allowing network
+validators to allocate resources to validating parachains. System chains are Polkadot using its
+scaling technology to host itself.
+
+See this
+[Polkadot blog article](https://polkadot.network/common-good-parachains-an-introduction-to-governance-allocated-parachain-slots/)
+and this
+[Polkadot Forum thread](https://forum.polkadot.network/t/polkadot-protocol-and-common-good-parachains/866)
+for more information.
+
+## On-demand Parachains
+
+:::info On-demand parachains were previously named parathreads
+
+On-demand parachains (previously called parathreads) are parachains that acquired
+[on-demand coretime](./learn-agile-coretime.md#on-demand-coretime).
+
+:::
+
+On-demand parachains temporarily participate (on a block by block basis) in
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} security without needing to lease a
+dedicated parachain slot. This is done through economically sharing the scarce resource of a
+_parachain slot_ (or core) among several competing resources (parachains). Chains that otherwise
+would not be able to acquire a full parachain slot or do not find it economically sensible to do so,
+can participate in {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}'s shared security
+as the [on-demand coretime](./learn-agile-coretime.md#on-demand-coretime) offers a graceful off-ramp
+to parachains that no longer require a dedicated parachain slot, but would like to continue using
+the Relay Chain.
+
+### Origin
```

Comment:
Changing the section title to not be confused with Origins with priviliges (Origins and Tracks etc)

```suggestion
### Historical Context of On-demand parachains
```